documentation/src/docs/recovery-procedures/metadata-table-segment-recovery.md

```diff
@@ -0,0 +1,120 @@
+<!--
+Copyright Pravega Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+
+# Metadata Table Segment Attribute Index Recovery
+
+
+The below outlined steps refer to recovery of Attribute Index Table Segments but only in cases of Metadata Segments which are a special case, as they need to
+be handled a little differently. Metadata Segments in Pravega are Table Segments, that hold metadata about all other "non-metadata" Segments, which
+include the internal Segments as well as the User Segments. Metadata about these Metadata Segments themselves in Pravega, are stored in Pravega Journals. More details
```

Comment:
`in Pravega,` -> can be removed. Also, are they called Pravega Journals or System Journals?

Reply:
Fixed. changed to Pravega System Journals.

Reply:
I think that there is another mention to Journals or Pravega Journals, not System Journals.